gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineExactDependencies.java

```diff
@@ -199,7 +198,7 @@ private static void configureSourceSet(
         checkUnusedDependencies.configure(task -> task.dependsOn(sourceSetUnusedDependencies));
         TaskProvider<CheckImplicitDependenciesTask> sourceSetCheckImplicitDependencies = project.getTasks()
                 .register(
-                        GUtil.toLowerCamelCase("checkImplicitDependencies " + sourceSet.getName()),
```

Comment:
GUtil is deprecated but usually source sets should already use camel case so this should be equivalent.